### ISSUE TITLE
Add support for Slide Fragments in editor

### ISSUE THREAD

**Issue by bwulff** [slidewiki/slidewiki-platform#1163]
reveal.js supports the ```fragment``` class for initially hiding elements on a slide and then making them visible step by step. See: https://revealjs.com/#/fragments

This could be a nice enhancement if we expose the feature in the editor since it would enable users to produce more dynamic and structured presentations while not changing the markup significantly which means that it wouldn't affect any content analysis.

Enhancement was proposed by a user via feedback form.

**Comment by kadevgraaf** [comment]
👍 

I proposed and discussed this in SWIK-1778 - https://slidewiki.atlassian.net/browse/SWIK-1778 but we never got around to implement it.

@aoelen do you think this is a worthwhile feature to work on?

From the discussion:
> A possible approach would be to append the context menu with options to apply the different fragments to the input box that the context menu controles. Downside: no preview.
> Another option would be to have a preview of possible fragments in the slide-edit left menu - after clicking a fragment the user would also have to click an element on the slide to apply the fragment to. Downside: more actions and possibly less intuitive for users.

**Comment by aoelen** [comment]
I think it is a nice feature, however with a low priority. As you mentioned, the potential problem with this feature would be extra complexity for the users. At this moment, I think it is more important to make the slide editor itself more intuitive, instead of adding new features to it.

**Comment by bwulff** [comment]
I agree that there is rather complex user interaction involved with this and it is definitely low prio. Just wanted to file it so that we have it on the radar (wasn't aware of SWIK-1778).
